### PR TITLE
Add media type binary

### PR DESCRIPTION
**Pull request by elocremarc**

### CODE REVIEW

File: src/media.rs [hunk 1]
```diff
@@ -26,6 +26,7 @@ impl Media {
     ("application/pgp-signature", Media::Text, &["asc"]),
     ("application/protobuf", Media::Unknown, &["binpb"]),
     ("application/yaml", Media::Code, &["yaml", "yml"]),
+    ("application/octet-stream", Media::Unknown, &[".bin"]),
```
Comment (raphjaph) [inline src/media.rs:29]:
```suggestion
    ("application/octet-stream", Media::Unknown, &["bin"]),
```

**Comment by raphjaph** [comment]
The use case would be chunking inscription data/content across many different inscriptions and then loading them recursively?

**Comment by raphjaph** [comment]
Could you merge and resolve the conflicts?

**Comment by raphjaph** [comment]
I can do it quickly

**Comment by elocremarc** [comment]
> 
Could you merge and resolve the conflicts?

Done just to confirm its BROTLI_MODE_GENERIC right?
```rust
    ("application/octet-stream",    BROTLI_MODE_GENERIC, Media::Unknown,  &["bin"]),
```

**Comment by elocremarc** [comment]
ahaha lint. We gotta get my clippy working lol

**Review by raphjaph — APPROVED** [review]

**Pull request merged**